data/autoscale-formula/SettingInitialPoolSize.txt

```diff
@@ -0,0 +1,6 @@
+$TargetDedicated = 4;
+lifespan = time() - time(\"Mon, 06 Oct 2014 10:20:00 GMT\");
```

Comment:
"Mon, 06 Oct 2014 10:20:00 GMT" is supposed to indicate datetime now. I think we should mention that here too. @timotheeguerin